tests/e2e/util/istioUtils.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+
+	"github.com/golang/glog"
+)
+
```

Comment:
Could you move hte istioctl related flags here and create a struct like
type istioctl struct {
remotePath string
binaryPath string
}

func (i *istioctl) DownloadIstioctl()

func (i *istioctl) KubeInject(yamlFile, svcName, yamlDir, proxyHub, proxyTag, namespace string) (string, error)